params/params_Madina.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 2
+    
+# e.g. "anas_krystallinus"    
+species.name <- "Laimoon "
```

Comment:
I think there might be an extra space here.

```suggestion
species.name <- "Laimoon"
```